source/includes/_orders.md

```diff
@@ -75,27 +95,27 @@ The Orders API returns order, garment, order status, and delay status objects.
-ordered_at | datetime | Time that the dealer completed the checkout process and officially placed the order
-created_at | datetime | Time when the dealer first began adding garments to the order
-invoiced_at | datetime | Time of the first invoice
-currency | subresource | currency used in the order
+| Attribute                                    | Description                                                                                                                                                                                                           |
+| -------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| id <br> <span>string</span>                  | Unique identifier for the object                                                                                                                                                                                      |
+| title <br> <span>string</span>               | Our SKU field for orders                                                                                                                                                                                              |
+| custom_order_number <br> <span>string</span> | Dealers can set this field to whatever they want.  It is typically the dealer's SKU or a summary of the order                                                                                                         |
+| garment_count <br> <span>integer</span>      | How many garments are in the order. multi-piece garments (e.g., Suit) are counted as 1.                                                                                                                               |
+| ship_type <br> <span>string</span>           | How is the garment shipped to the final destination                                                                                                                                                                   |
+| ship_cost <br> <span>decimal</span>          | How much did it cost to ship the whole order                                                                                                                                                                          |
+| subtotal <br> <span>decimal</span>           | Order amount before tax and discounts are included                                                                                                                                                                    |
+| dealer_discount <br> <span>decimal</span>    | *Description TBD*                                                                                                                                                                                                     |
```

Comment:
Found it: line 345 of pricing says: `dealer_discount is the amount of dealer's permanent discounts`